package.json

```diff
@@ -132,7 +132,7 @@
       "post-checkout": "yarn",
       "post-merge": "yarn",
       "post-rebase": "yarn",
-      "pre-push": "yarn lint && pretty-quick --staged lint-staged",
+      "pre-push": "yarn lint-fix && pretty-quick --staged lint-staged",
```

Comment:
This should be reverted as fixes will not be committed and will cause checks to fail. I didn't change this on purpose.

Reply:
good catch, I didn't want to push this, was just testing 😂

Reply:
fixed